### ISSUE TITLE
Input check storeId

### ISSUE THREAD

**Issue by patrick-bigbridge** [olegkoval/magento2-regenerate_url_rewrites#5]
I made a stupid mistake. I tried to regenerate for a single store view:
`bin/magento ok:urlrewrites:regenerate --storeId 19`
That didn't work, so I tried
`bin/magento ok:urlrewrites:regenerate storeId:19`
That didn't work either, so finally I performed the right command:
`bin/magento ok:urlrewrites:regenerate 19`

But in the mean while, the input 'storeId:19' had removed all url_rewrite data(!) I would have like to see the error that the store view did not exist, but that didn't happen because of this check:
`$storeId > 0` in line 92. 'storeId:19' is cast to 0 in the comparison.

Can you please adjust this check for fools like me?

**Comment by olegkoval** [comment]
Hello Patrick.

Thank you for submitting issue.

Could you exec this query in your Magento database and put results here (query will select list of all stores from your Magento instance)?
`select * from store;`

Because if store not exists (even if storeId > 0) then extension displays a error message and stop work, see lines 93 - 101:
`if (isset($allStores[$storeId])) {`
`    $storesList = array(`
`        $storeId => $allStores[$storeId]`
`    );`
`} else {`
`    $output->writeln('ERROR: store with this ID not exists.');`
`    $output->writeln('Finished');`
`    return;`
`}`

Best regards,
Oleg Koval

**Comment by patrick-bigbridge** [comment]
Here is the result:

```
+----------+---------+------------+----------+-------------+------------+-----------+
| store_id | code    | website_id | group_id | name        | sort_order | is_active |
+----------+---------+------------+----------+-------------+------------+-----------+
|        0 | admin   |          0 |        0 | Admin       |          0 |         1 |
|        4 | nl      |          1 |        1 | Netherlands |          0 |         1 |
|       14 | chstore |          3 |        5 | Zwitserland |          1 |         1 |
|       15 | bestore |          4 |        6 | Belgium     |          1 |         1 |
|       16 | enstore |          5 |        7 | English     |          1 |         1 |
|       18 | destore |          7 |        9 | German      |          1 |         1 |
|       19 | itstore |          8 |       10 | Italian     |          1 |         1 |
|       22 | atstore |         11 |       13 | Austria     |          1 |         1 |
|       23 | dkstore |         12 |       14 | Denmark     |          1 |         1 |
+----------+---------+------------+----------+-------------+------------+-----------+

```

What goes wrong is that the variable $storeId contains the string  "storeId:19" in my case. When it is compared to 0, as in $storeId > 0, $storeId is cast to an integer, and becomes 0. Since (0 > 0) is false, the check if the store id exists, is skipped entirely.

**Comment by olegkoval** [comment]
Hello Patrick.

I've added "fix" and released v1.0.4 with this fix.

Thank you.

Best regards,
Oleg Koval.

**Comment by patrick-bigbridge** [comment]
Thank you!
